test/unit/argument_parser/detail/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-add_definitions(-DSEQAN_INCLUDE_DIR="${SEQAN3_INCLUDE_DIR}")
+add_definitions(-DSEQAN3_TEST_LICENSE_DIR="${seqan3_test_unit_BINARY_DIR}/..")
```

Comment:
That means this path is somehow wrong

Reply:
Can you try this version?

```patch
diff --git a/test/unit/argument_parser/detail/CMakeLists.txt b/test/unit/argument_parser/detail/CMakeLists.txt
index 6f116f295..060a8dcb8 100644
--- a/test/unit/argument_parser/detail/CMakeLists.txt
+++ b/test/unit/argument_parser/detail/CMakeLists.txt
@@ -1,4 +1,7 @@
-add_definitions(-DSEQAN_INCLUDE_DIR="${SEQAN3_INCLUDE_DIR}")
+# seqan3_test_unit_BINARY_DIR is needed for debian builds
+find_path (SEQAN3_TEST_LICENSE_DIR NAMES LICENSE.md HINTS "${SEQAN3_ROOT_DIR}" "${seqan3_test_unit_BINARY_DIR}/..")
+
+add_definitions(-DSEQAN_INCLUDE_DIR="${SEQAN3_TEST_LICENSE_DIR}")
 seqan3_test(format_help_test.cpp)
 seqan3_test(format_html_test.cpp)
 seqan3_test(format_man_test.cpp)
```

Reply:
I still don't understand the file layout of debian.

Reply:
When Seqan3 is installed, `SEQAN_INCLUDE_DIR` is `/usr/include/seqan3/` or `/usr/include` (I forget which). Either way, adding on "../LICENSE.md" won't work

Reply:
Okay, but where is `LICENSE.md` located?